### ISSUE TITLE
Absolute path as `jsonnet_path` does not work

### ISSUE THREAD

**Issue by lohrm-stabl** [alxrem/terraform-provider-jsonnet#9]
I have the following TF json:
<details><summary>Details</summary>
<code>
{
  "//": {
    "metadata": {
      "backend": "local",
      "stackName": "infra",
      "version": "0.19.0"
    },
    "outputs": {
    }
  },
  "data": {
    "jsonnet_file": {
      "jsonnet-dashboard": {
        "//": {
          "metadata": {
            "path": "infra/jsonnet-dashboard",
            "uniqueId": "jsonnet-dashboard"
          }
        },
        "source": "/workspaces/Grafana-Infrastructure/dashboards/3ph-generic.libsonnet"
      }
    }
  },
  "provider": {
    "grafana": [
      {
      }
    ],
    "jsonnet": [
      {
        "jsonnet_path": "/workspaces/Grafana-Infrastructure/dashboards/vendor"
      }
    ]
  },
  "resource": {
    "grafana_dashboard": {
      "grafana-dahsboard": {
        "//": {
          "metadata": {
            "path": "infra/grafana-dahsboard",
            "uniqueId": "grafana-dahsboard"
          }
        },
        "config_json": "${data.jsonnet_file.jsonnet-dashboard.rendered}",
        "message": "Automatically generated by Grafanonet and CDKTF",
        "overwrite": true
      }
    }
  },
  "terraform": {
    "backend": {
      "local": {
        "path": "/workspaces/Grafana-Infrastructure/infrastructure/terraform.infra.tfstate"
      }
    },
    "required_providers": {
      "grafana": {
        "source": "grafana/grafana",
        "version": "2.3.3"
      },
      "jsonnet": {
        "source": "alxrem/jsonnet",
        "version": "2.3.0"
      }
    }
  }
}
</code>
</details>

This does not work:
```
       │ RUNTIME ERROR: couldn't open import
       │ "github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet": no match
       │ locally or in the Jsonnet library paths
       │        /workspaces/Grafana-Infrastructure/dashboards/grafana.libsonnet:1:1-74  $
       │        /workspaces/Grafana-Infrastructure/dashboards/3ph-generic.libsonnet:1:11-39
       │ thunk <g> from <$>
       │        /workspaces/Grafana-Infrastructure/dashboards/3ph-generic.libsonnet:8:1-2
       │        During evaluation
```

But if i remove `"jsonnet_path": "/workspaces/Grafana-Infrastructure/dashboards/vendor"` and set it via env instead:
`JSONNET_PATH=/workspaces/Grafana-Infrastructure/dashboards/vendor terraform apply`
It works fine.

But it should not matter if set via TF or env, right?

**Comment by alxrem** [comment]
Hello. Thank you for report.
This issue should be fixed in the version v2.3.1

**Comment by lohrm-stabl** [comment]
Can confirm. Thanks for the quick fix!
